lib/recurly/pager.php

```diff
@@ -45,15 +44,14 @@ public function rewind() {
    */
   public function current()
   {
-    if (empty($this->_count)) {
+    if ($this->count() == 0) {
```

Comment:
If that's the case, perhaps one or two well named methods on the Pager class would be good in this situation for readability. As it stands it kind of looks like you just forgot to use empty. I couldn't think of what those names would be off the top of my head. I just couldn't tell that as a reader there would be a distinction between 0 and null.